pkgs/by-name/fi/filen-desktop/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  appimageTools,
+  makeDesktopItem,
+}:
+let
+  pname = "filen-desktop";
+  version = "3.0.41";
+
+  arch = builtins.head (builtins.split "-" stdenv.hostPlatform.system);
+
+  src = fetchurl {
+    url = "https://github.com/FilenCloudDienste/filen-desktop/releases/download/v${version}/Filen_linux_${arch}.AppImage";
+    sha256 = "sha256-Nao5By8Z8lMbRcp2Mgw+xaiiFzUxCm6S3SAE5FfDZpk=";
```

```suggestion
    hash = "sha256-Nao5By8Z8lMbRcp2Mgw+xaiiFzUxCm6S3SAE5FfDZpk=";
```

please use `hash` instead of `sha256`.